bibharvest/bibfilter_oaipos2inspire.py

```diff
@@ -101,7 +102,11 @@ def main(args):
 
         url = base_url + identifier
         session = requests.session()
-        r = session.get(url)
+        try:
+            r = session.get(url, timeout=60)
+        except ConnectionError, Timeout:
+            register_exception()
+            continue
```

Comment:
@kaplun   good question
the way I see it the metadata is already harvested and this filter step attempts to attach fulltext one record at a time by screen-scraping a corresponding web-page. If that fails, I don't think the process should fail completely and require restarting everything. I guess adding 
`error_records.append(rec)`
which is already used when screenscraping fails to find a pdf would make sense, though.